src/main/scala/wasm/MiniWasm.scala

```diff
@@ -301,7 +330,12 @@ object Evaluator {
         val frameLocals = args ++ locals.map(_ => I32V(0)) // GW: always I32? or depending on their types?
         val newFrame = Frame(frame.module, ArrayBuffer(frameLocals: _*))
         val newK: Cont[Ans] = (retStack) =>
-          eval(rest, retStack.take(ty.out.size) ++ newStack, frame, kont, trail, ret)
+          eval(rest,
```

Comment:
Nit: I would prefer not wrap here (and similarly for other `eval` calls), maybe increase the margin of your formatted? :)